### PR TITLE
[MWPW-148199] Accessible Radio Buttons

### PR DESCRIPTION
**Pull request by echen-adobe**
Describe your specific features or fixes

Makes radio buttons accessible without changing dom structure

Resolves: https://jira.corp.adobe.com/browse/MWPW-148199

Test URLs:
- Before: https://main--express--adobecom.hlx.page/express/
- After: https://accessible-radio-buttons--express--adobecom.hlx.page/express/pricing?martech=off

### CODE REVIEW

File: express/blocks/billing-radio/billing-radio.js [hunk 1]
```diff
@@ -11,25 +10,79 @@ export default function init(el) {
...
+        e.preventDefault();
+        focusNextButton(currentIndex);
+        break;
+      case 'Enter':
+      case 'Space':
+        e.preventDefault();
+        BlockMediator.set(BILLING_PLAN, currentIndex);
+        break;
+      case 'Tab':
+        console.log(el, el.nextElementSibling);
```
Comment (JingleH) [inline express/blocks/billing-radio/billing-radio.js:72]: the console log can be removed

File: express/blocks/billing-radio/billing-radio.js [hunk 2]
```diff
@@ -11,25 +10,79 @@ export default function init(el) {
...
+      case 'ArrowDown':
+        e.preventDefault();
+        focusNextButton(currentIndex);
+        break;
+      case 'Enter':
+      case 'Space':
+        e.preventDefault();
+        BlockMediator.set(BILLING_PLAN, currentIndex);
+        break;
+      case 'Tab':
```
Comment (JingleH) [inline express/blocks/billing-radio/billing-radio.js:71]: i gave a try and I think the tabbing is disabled by the tabIndex==-1. Is this expected?

**Review by JingleH — CHANGES_REQUESTED** [review]

File: express/blocks/billing-radio/billing-radio.js (hunk 2 above)
Comment (echen-adobe) [reply express/blocks/billing-radio/billing-radio.js:71]: That is intended behavior. To fully mimic regular radio buttons the arrow keys should be the ones that navigate the control. The tab key should instead bring the user to the next element outside of the billing radio group.

File: express/blocks/billing-radio/billing-radio.js (hunk 1 above)
Comment (echen-adobe) [reply express/blocks/billing-radio/billing-radio.js:72]: Removed just now

**Review by qiyundai — APPROVED** [review]

**Pull request merged**